compiler-rt/lib/asan/asan_report.cpp

```diff
@@ -34,8 +34,8 @@ namespace __asan {
 // -------------------- User-specified callbacks ----------------- {{{1
 static void (*error_report_callback)(const char*);
 using ErrorMessageBuffer = InternalMmapVectorNoCtor<char, true>;
-static ALIGNED(
-    alignof(ErrorMessageBuffer)) char error_message_buffer_placeholder
+alignas(
```

Comment:
is reordering static and align is necessary,
I realy like static being the first.

The same for the rest.

Reply:
`alignas` (attribute specifier) must precede `static` (storage class specifier) in the decl-specifier.  Both Clang and GCC have errors if the order is swapped.

I think the relevant rules are here:

https://eel.is/c++draft/dcl.pre#nt:simple-declaration specifies the order: [attribute-specifier-seq](https://eel.is/c++draft/dcl.attr.grammar#nt:attribute-specifier-seq) [decl-specifier-seq](https://eel.is/c++draft/dcl.spec.general#nt:decl-specifier-seq) [init-declarator-list](https://eel.is/c++draft/dcl.decl.general#nt:init-declarator-list) ;